core/templates/vector.h

```diff
@@ -253,6 +250,13 @@ class Vector {
 
 	_FORCE_INLINE_ Vector() {}
 	_FORCE_INLINE_ Vector(const Vector &p_from) { _cowdata._ref(p_from._cowdata); }
+	_FORCE_INLINE_ Vector(const uint8_t *p_from, int32_t p_size) {
```

Comment:
I don't think this function is a good idea, will never be used as often to justify it. Just memcpy manually to it.